sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -0,0 +1,171 @@
+# Azure Form Recognizer client library for Python
+
+* TODO
+* TODO
+
+[Source code][TODO] | [Package (PyPI)][TODO] | [API reference documentation][TODO]| [Product documentation][TODO] | [Samples][TODO]
+
+## Getting started
+
+### Prerequisites
+* Python 2.7, or 3.5 or later is required to use this package.
+* You must have an [Azure subscription][azure_subscription] and a
+[Cognitive Services or Form Recognizer resource][FR_or_CS_resource] to use this package.
+
+### Install the package
+Install the Azure Form Recognizer client library for Python with [pip][pip]:
+
+```bash
+pip install azure-ai-formrecognizer --pre
```

Comment:
Discussed with @johanste we should not have `--pre` if the package has never been released in stable yet (only preview).

Reply:
Was wondering about this, I removed it